packages/framework/src/schemas/steps/channels/in-app.schema.ts

```diff
@@ -1,14 +1,47 @@
-import { Schema } from '../../../types/schema.types';
+import { ExtendedJsonSchema, Schema } from '../../../types/schema.types';
+
+const ABSOLUTE_AND_RELATIVE_URL_REGEX = '^(?!mailto:)(?:(https?):\\/\\/[^\\s/$.?#].[^\\s]*)|^(\\/[^\\s]*)$';
```

Comment:
Since we're introducing the `mailto` URL directive here, should we also include the `tel` directive ([ref](https://stackoverflow.com/a/3864106))?